### PR TITLE
[PDMO-88] Improve UI/UX for date range picker

### PR DESCRIPTION
**Pull request by Farhanisto**
## JIRA TICKET NAME:

[PDMO-88 Improve UI/UX for date range picker](https://issues.openmrs.org/browse/PDMO-88)

## SUMMARY:
currently, the date range picker is displayed vertically which needs to be changed to horizontal view for better UX 
current the ranges are in text form which makes it hard to spot, they should also be re-designed to resemble a link.

### CODE REVIEW

File: app/css/obsadmin.css [hunk 1]
```diff
@@ -857,3 +857,272 @@ input[type="radio"]{
...
+  position: absolute;
+  color: inherit;
+  background-color: #fff;
+  border-radius: 4px;
+  width: 278px;
+  padding: 4px;
+  margin-top: 1px;
+  top: 100px;
+  left: 20px;
+  /* Calendars */ }
```
Comment (mkiterian) [inline app/css/obsadmin.css]: Should the `Calendars` comment be here?

**Comment by mkiterian** [comment]
The indentation in the css file does not seem to be consistent

**Comment by Farhanisto** [comment]
@mkiterian thanks for the pointers .I have fixed

File: app/css/obsadmin.css [hunk 2]
```diff
@@ -857,3 +857,271 @@ input[type="radio"]{
...
+  padding: 4px;
+  margin-top: 1px;
+  top: 100px;
+  left: 20px;
+   }
+.daterangepicker:before, .daterangepicker:after {
+  position: absolute;
+  display: inline-block;
+  border-bottom-color: rgba(0, 0, 0, 0.2);
+  content: ''; }
```
Comment (brayoh) [inline app/css/obsadmin.css]: the closing bracket should on it's own line

File: app/css/obsadmin.css [hunk 3]
```diff
@@ -857,3 +857,271 @@ input[type="radio"]{
...
+.daterangepicker:before {
+  top: -7px;
+  border-right: 7px solid transparent;
+  border-left: 7px solid transparent;
+  border-bottom: 7px solid #ccc; }
+.daterangepicker:after {
+  top: -6px;
+  border-right: 6px solid transparent;
+  border-bottom: 6px solid #fff;
+  border-left: 6px solid transparent; }
```
Comment (brayoh) [inline app/css/obsadmin.css]: same here and the rest of the file

**Comment by Farhanisto** [comment]
Thanks @brayoh ...i have fixed

**Pull request merged**